addons/stock/models/stock_move.py

```diff
@@ -965,6 +966,7 @@ def _prepare_procurement_values(self):
             elif self.rule_id.group_propagation_option == 'none':
                 group_id = False
         return {
+            'description': self.description or self.name.replace(self.product_id.display_name, ''),
```

Comment:
why not use description_picking here?

Reply:
when duplicate the SO(which has custom detail in SOline) and confirm it then description picking will have only product name without custom detail.